pkgs/applications/networking/cluster/k3s/builder.nix

```diff
@@ -184,6 +185,16 @@ let
     src = k3sRepo;
     vendorHash = k3sVendorHash;
 
+    patches =
+      # Disable: Add runtime checking of golang version
```

Comment:
What problem did you encounter that requires removal of the runtime check for go version?